static/skywire-manager-src/src/assets/i18n/en.json

```diff
@@ -131,14 +131,15 @@
   "nodes": {
     "title": "Visor list",
     "dmsg-title": "DMSG",
-    "update-hypervisor": "Update hypervisor",
     "update-all": "Update all visors",
+    "hypervisor": "Hypervisor",
     "state": "State",
     "state-tooltip": "Current state",
     "label": "Label",
     "key": "Key",
     "dmsg-server": "DMSG server",
     "ping": "Ping",
+    "hypervisor-info": "The visor contains the currently used Hypervisor.",
```

Comment:
`This visor is the current Hypervisor.` sounds better. Other than that good job.